tests/hikari/impl/test_interaction_server.py

```diff
@@ -115,6 +155,19 @@ def test___init___without_public_key(self, mock_verifier, mock_rest_client, mock
 
         assert result._verify is None
 
+    @pytest.mark.parametrize(
+        "key",
+        [
+            "okokokokokokokokokokokokokokokok",
+            b"NO",
+        ],
+    )
+    def test___init___with_invalid_public_key(self, key: typing.Union[str, bytes]):
+        with mock.patch.object(aiohttp.web, "Application"):
+            interaction_server_impl.InteractionServer(
+                dumps=object(), entity_factory=object(), loads=object(), rest_client=object(), public_key=key
+            )
```

Comment:
There are no checks here. Is this test incomplete?

Reply:
the tests haven't been done yet